src/Warning/Warning.js

```diff
@@ -0,0 +1,271 @@
+/* @flow */
+
+import * as React from 'react';
+import { Animated, View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { getStatusBarHeight } from 'react-native-status-bar-height';
+
+import { Icon } from '../Icon';
+import Touchable from '../Button/Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+import type { OnLayout, NotificationType } from '../types';
+
+import Text from '../Text';
+
+type Props = {|
+  +style?: StylePropType,
+  +onDismiss: () => void,
+|};
+
+type State = {|
+  layout: {
+    height: number,
+    measured: boolean,
+  },
+  opacity: Animated.Value,
+  translateY: Animated.Value,
+  isOpen: boolean,
+  warningType: NotificationType,
+  warningTitle: React.Node | string,
+  warningMessage: React.Node | string,
+|};
```

Comment:
I think you can unify types with Alerts. Have some `type AlertBase` and then do `AlertBase & WarningStateType`.

Reply:
Will do that.